.circleci/config.yml

```diff
@@ -1374,6 +1427,17 @@ workflows:
               gpus-per-machine: [4]
               num-machines: [2]
 
+      - test-det-deploy:
+          name: test-det-deploy-local
+          requires:
+            - package-and-push-system-local
+          matrix:
+            parameters:
+              parallelism: [2]
+              mark: ["det_deploy_local"]
+              det-version: [$CIRCLE_SHA1]
+
```

Comment:
Should this have a `filter: branches` on it so it only runs on the master branch?